pkg/project/types/types.go

```diff
@@ -33,10 +33,13 @@ type Build struct {
 }
 
 type Project struct {
-	Name      string    `toml:"name"`
-	Version   string    `toml:"version"`
-	SourceURL string    `toml:"source-url"`
-	Licenses  []License `toml:"licenses"`
+	ID               string    `toml:"id"`
+	Name             string    `toml:"name"`
+	Version          string    `toml:"version"`
+	Authors          []string  `toml:"authors"`
+	DocumentationURL string    `toml:"documentation-url"`
+	SourceURL        string    `toml:"source-url"`
+	Licenses         []License `toml:"licenses"`
```

Comment:
According to https://github.com/buildpacks/spec/blob/extensions/project-descriptor/0.1/extensions/project-descriptor.md#project